core/util/datetime_helpers.py

```diff
@@ -60,3 +61,14 @@ def strptime_utc(date_string, format):
     if "%Z" in format or "%z" in format:
         raise ValueError(f"Cannot use strptime_utc with timezone-aware format {format}")
     return to_utc(datetime.datetime.strptime(date_string, format))
+
+
+def previous_months(number_of_months=3) -> Tuple[datetime.date, datetime.date]:
```

Comment:
Copypasta? The `number_of_months` parameter should probably not have a default value.

Reply:
Copypasta is a synonym for coding